frontend/src/app/libs/landing/services/event-converter.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { Event } from '../models/event';
+import { EventDto } from '../models/eventDto';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class EventConverter {
+	public convertFromDto(eventsDto: EventDto[]): Event[] {
+		return eventsDto.map(({ _id, tasks, participations, maxScore, isActive, users, ...dto }: EventDto) => ({ id: _id, ...dto }));
```

Comment:
please simplify `({ _id, tasks, participations, maxScore, isActive, users, ...dto }` to `{ _id, ...dto}` since we are using only _id and dto properties